unit/av_test.c

```diff
@@ -1036,6 +1040,7 @@ int main(int argc, char **argv)
 			printf("\t[-a fabric_name]\n");
 			printf("\t[-n num_good_addr (max=%d)]\n", MAX_ADDR - 1);
 			printf("\t[-f provider_name]\n");
+			printf("\t[-s source_address]\n");
```

Comment:
Future TODO note: convert this arg parsing to the common scheme found in the other tests (eg pingpong/msg_pingpong.c).